### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by mugnap** [EvoMobius/OneSignal.net#1]
Hello, I like it and it works, but when I pass a WRONG PlayerID, the method 
`.notificationService.Create(createOptions) `
crashes and returns the following error:

```
Newtonsoft.Json.JsonSerializationException: 'Cannot deserialize the current JSON array (e.g. [1,2,3]) into type 'OneSignal.CreateNotificationErrors' because the type requires a JSON object (e.g. {"name":"value"}) to deserialize correctly.
To fix this error either change the JSON to a JSON object (e.g. {"name":"value"}) or change the deserialized type to an array or a type that implements a collection interface (e.g. ICollection, IList) like List<T> that can be deserialized from a JSON array. JsonArrayAttribute can also be added to the type to force it to deserialize from a JSON array.
Path 'errors', line 1, position 34.'
```

I'm not expert to debug it but I think that it happens because OneSignal can also return an error without **invalid_player_ids** node (see https://documentation.onesignal.com/reference/create-notification at the bottom of the page, **200 No Subscribed Players**:

```
{
    "id": "",
    "recipients": 0,
    "errors": [
        "All included players are not subscribed"
    ]
}
```
Can you solve it? Thanks!

**Comment by donald-slagle** [comment]
Sorry for the late reply. I'll get this sorted out today and get a new release out.

**Comment by donald-slagle** [comment]
This issue is now fixed in version 1.0.6. A new property named `ErrorMessages` has been added to the `CreateNotificationErrors` object that will contain non player id error messages.
